src/vector_of_array.jl

```diff
@@ -721,6 +724,18 @@ end
     VectorOfArray([similar(VA[:, i], T) for i in eachindex(VA.u)])
 end
 
+# for VectorOfArray with multi-dimensional parent arrays of arrays where all elements are the same type
+@inline function Base.similar(vec::VectorOfArray{
```

Comment:
```suggestion
function Base.similar(vec::VectorOfArray{
```

Is that well-tested? What's the reason for overriding the inlining calculation?

Reply:
Yeah, it's tested. 

I just copied the `@inline` from before, I didn't think about it overriding the other inlining.

Reply:
Thanks!